packages/hledger/build.sh

```diff
@@ -0,0 +1,29 @@
+TERMUX_PKG_HOMEPAGE=https://hledger.org/
+TERMUX_PKG_DESCRIPTION="Robust, friendly, fast plain text accounting software"
+TERMUX_PKG_LICENSE="GPL-3.0"
```

Comment:
If you want to be more specific you can specify

```suggestion
TERMUX_PKG_LICENSE="GPL-3.0-or-later"
```

See: https://github.com/simonmichael/hledger/blob/7482f1efc37d8d763f2ea0d7038da00d7032f4e5/hledger/hledger.cabal#L26